core/src/test/java/com/g16/feyrune/model/overworld/map/TestMapManager.java

```diff
@@ -0,0 +1,91 @@
+package com.g16.feyrune.model.overworld.map;
+
+import org.junit.Test;
+
+import java.awt.*;
+
+import static org.junit.Assert.*;
+
+public class TestMapManager {
```

Comment:
This file does no longer exist in main, and related functionality has been moved to OverworldModel.